_aicspylibczi/exceptions.h

```diff
@@ -131,6 +131,18 @@ class CdimSelectionZeroImagesException : public std::runtime_error
   }
 };
 
+class StrideAssumptionExcetpion : public std::runtime_error
+{
+public:
+  explicit StrideAssumptionExcetpion(const std::string& message_)
+    : std::runtime_error("Image Stride % Width != 0. Please create an issue at \n"
```

Comment:
this statement is wrong i think... it gets thrown when stride<width, not the modulo thing you wrote here?

Reply:
Exception is also misspelled here too!

Reply:
That said. I love this error message.

Reply:
It should be thrown when Stride % width != 0 and Stride < width. That implies some form of compression was used which wasn't accounted for or the image is corrupted.